src/redux/actionCreators.js

```diff
@@ -87,12 +92,12 @@ const getUnfollowThunkCreator = (u) => {
   };
 };
 
-const getAuthUserData = () => (dispatch) => {
-  usersAPI.authMe().then((response) => {
-    if (response.data.resultCode === 0) {
-      dispatch(setUserData(response.data.data));
-    }
-  });
+const getAuthUserData = () => async (dispatch) => {
```

Comment:
Consider moving these thunks to the thunks.js.

Reply:
done